addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1694,6 +1694,21 @@ body.editor_enable.editor_has_snippets {
                 background-color: $o-we-sidebar-bg;
             }
         }
+
+        div.o_we_invisible_root_parent {
+            padding-bottom: 3px;
+        }
+
+        ul {
+            list-style: none;
+            padding-inline-start: 15px;
+            margin-bottom: $o-we-sidebar-content-field-spacing - 3px;
+
+            div.o_we_invisible_entry {
+                padding-top: 3px;
+                padding-bottom: 3px;
+            }
```

Comment:
`o_we_invisible_root_parent` elements are always `o_we_invisible_entry` right ?
Can't you get rid of `o_we_invisible_root_parent` by doing something like that ?
```scss
div.o_we_invisible_entry:has(+ ul) {
            padding-bottom: 3px;
}
```

Reply:
My comment is wrong as `:has()` is not well suported :+1: